tools: update dashboard content (#1073)
pingcap/docs: tools/dm/monitor.md

## Instance

In the Grafana dashboard, the default name of an instance is `DM-instance`.

### Relay log

| Metric name | Description | Alert |
|:----|:------------|:----|
| storage capacity | The total storage capacity of the disk occupied by the relay log  | N/A |
| storage remain | The remaining storage capacity within the disk occupied by the relay log | An alert is needed once the value is smaller than 10G |
| process exits with error | The relay log encounters an error in DM-worker and exits  | Immediate alerts |
| relay log data corruption | The number of corrupted relay logs | Immediate alerts |
| fail to read binlog from master | The number of errors encountered when relay log reads the binlog from the upstream MySQL | Immediate alerts |
| fail to write relay log | The number of errors encountered when the relay log writes the binlog to disks | Immediate alerts |
| binlog file index | The largest index number of relay log files. For example, "value = 1" indicates "relay-log.000001" | N/A |

> | binlog file gap between master and relay | The number of binlog files in the relay log that are behind the upstream master | When the number of binlog files in relay that are behind the upstream master exceeds one (>1) and the condition lasts over 10 minutes |

Reviewer: ```suggestion
| binlog file gap between master and relay | The number of binlog files in the relay log that are behind the upstream master | An alert occurs when the number of binlog files in relay that fall behind the upstream master exceeds one (>1) and the condition lasts over 10 minutes |
```

| binlog pos | The write offset of the latest relay log file  | N/A |
| read binlog duration | The duration that the relay log reads the binlog from the upstream MySQL (in seconds) |  N/A |
| write relay log duration | The duration that the relay log writes the binlog into the disk each time (in seconds) | N/A |
| binlog size | The size of a single binlog event that the relay log writes into the disks | N/A |

### Task

| Metric name | Description | Alert |
|:----|:------------|:----|
| task state | The state of subtasks for replication | An alert occurs when the subtask has been paused for more than 10 minutes |
| load progress | The percentage of loading completion of loader. The value is between 0% and 100%  | N/A |

> | binlog file gap between master and syncer | The number of binlog files by which binlog replication is behind the upstream master  | N/A |

Reviewer: ```suggestion
| binlog file gap between master and syncer | The number of binlog files in binlog replication that fall behind the upstream master  | N/A |
```
